index.html

```diff
@@ -0,0 +1,141 @@
+          <button type="button"class="responsive">JavaScript</button>
+        </div>
+          <button type="button" class="show-works">
+          See this work here &#x2192
+          </button>
+      </div>
+
+      <div class="project-2">
+        <img src="images/Rectangle 22.png" alt="" class="project2">
+        <div class="work-2">
+<h2 class="second-project">Project name goes here</h2>
+ <div class="languages-2">
+          <button type="button" class="mark-up-2">HTML/CSS</button>
+          <button type="button" class="react-2">Ruby on Rails</button>
+          <button type="button" class="responsive-2">JavaScript</button>
+        </div>
+        </div>
+          <button type="button" class="show-works-2">
+          See this work here &#x2192
+          </button>
+        </div>
+      </div>
+
+    <div class="project-3">
+      <img src="images/project 3.png" alt="" class="project3">
+      <div class="work-3">
+ <h2 class="third-project">Project name goes here</h2>
+<div class="languages-3">
+          <button type="button" class="mark-up-3">HTML/CSS</button>
+          <button type="button" class="react-3">Ruby on Rails</button>
+          <button type="button" class="responsive-3">JavaScript</button>
+        </div>
+      </div>
+      <button type="button" class="show-works-3">
+          See this work here &#x2192
+          </button>
+         </div>
+
+          <div class="project-4">
+      <img src="images/Project 4.png" alt="" class="project4">
+      <div class="work-4">
+ <h2 class="forth-project">Project name goes here</h2>
+<div class="languages-4">
+          <button type="button" class="mark-up-4">HTML/CSS</button>
+          <button type="button" class="react-4">Ruby on Rails</button>
+          <button type="button" class="responsive-4">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-4">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+<div class="project-5">
+      <img src="images/Project 5.png" alt="" class="project5">
+      <div class="work-5">
+ <h2 class="fifth-project">Project name goes here</h2>
+<div class="languages-5">
+          <button type="button" class="mark-up-5">HTML/CSS</button>
+          <button type="button" class="react-5">Ruby on Rails</button>
+          <button type="button" class="responsive-5">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-5">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+    <div class="project-6">
+      <img src="images/Project 6.png" alt="" class="project6">
+      <div class="work-6">
+ <h2 class="sixth-project">Project name goes here</h2>
+<div class="languages-6">
+          <button type="button" class="mark-up-6">HTML/CSS</button>
+          <button type="button" class="react-6">Ruby on Rails</button>
+          <button type="button" class="responsive-6">JavaScript</button>
+        </div>
```

Comment:
- [ OPTIONAL ] Languages should use a `p` tag and not `button` tag since they are not supposed to do any action. Please consider changing that
- [ OPTIONAL ] Also, they should be displayed in a list using a `ul` and each of them in a `li`